include/ttmlir/Dialect/TTIR/Pipelines/TTIRPipelines.h

```diff
@@ -17,9 +17,10 @@ struct StableHLOToTTIRPipelineOptions
   Option<bool> removeDeadValuesEnabled{
       *this, "enable-remove-dead-values",
       llvm::cl::desc("Enable --remove-dead-values optimization pass."),
-      // Currently this pass fails if module has a name, so keeping the
-      // optimization OFF by default until that issue is fixed on llvm side.
-      llvm::cl::init(false)};
+      llvm::cl::init(true)};
+  Option<bool> SparseConstantPropogationEnabled{
```

Comment:
Maybe lowercase variable name? I know it's common in LLVM to do this, but in tt-mlir it's not so we should have consistent style.